packages/compass-e2e-tests/index.js

```diff
@@ -4,16 +4,50 @@ const { promisify } = require('util');
 const glob = require('glob');
 const { sync: spawnSync } = require('cross-spawn');
 const Mocha = require('mocha');
+const { MongoClient } = require('mongodb');
 const debug = require('debug')('compass-e2e-tests');
 const {
   rebuildNativeModules,
   compileCompassAssets,
   buildCompass,
 } = require('./helpers/compass');
 const { createUnlockedKeychain } = require('./helpers/keychain');
+const DBLogger = require('./helpers/db-logger');
 
 const keychain = createUnlockedKeychain();
 
+let client;
+
+function getMetricsConnectionString() {
+  if (process.env.METRICS_CONNECTION_STRING) {
```

Comment:
Should we prefix the var with `E2E_` to be consistent with the rest of vars?

Btw, It wouldn't be that bad to use a dedicated variable / connection string for this also on the CI.

Reply:
I suppose then we don't need a fallback.

Reply:
Happy with `E2E_TESTS_METRICS_STRING`? Then that's the only way to have metrics stored in the db with no EE_ATLAS_* fallbacks.